lib/improver/cli/weighted_blending.py

```diff
@@ -176,27 +176,106 @@ def main(argv=None):
     if (args.wts_calc_method == "dict") and not args.wts_dict:
```

Comment:
Do these checks need to be moved entirely into the process ot keep the CLI check and add new checks inside process?